src/openrct2/peep/GuestPathfinding.cpp

```diff
@@ -2073,11 +2073,11 @@ int32_t guest_path_finding(Guest* peep)
 
         int16_t stationX = (int16_t)(entranceLocation.x * 32);
         int16_t stationY = (int16_t)(entranceLocation.y * 32);
-        uint16_t dist = abs(stationX - peep->next_x) + abs(stationY - peep->next_y);
-
-        if (dist < closestDist)
+        uint16_t score = abs(stationX - peep->next_x) + abs(stationY - peep->next_y);
```

Comment:
I think distance makes still more sense in this context than score, its looking for the smallest distance here.

Reply:
imho score matches the rest of the pathfinding file.